pkg/storage/accountsDB/accountInterface.go

```diff
@@ -0,0 +1,16 @@
+package accountsDB
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+)
+
+type AccountsService interface {
+	GetAccount(context.Context, uuid.UUID) (model.Account, error)
+	GetUserAccounts(context.Context, uuid.UUID) ([]model.Account, error)
+	GetAllUsers(context.Context) ([]model.Account, error)
+	CreateAccount(context.Context, uuid.UUID) (model.Account, error)
+	UpdateAccount(context.Context, model.Account) (model.Account, error)
+	DeleteAccount(context.Context, uuid.UUID) error
```

Comment:
Names can be same as at `User(Get, Update, etc)`

Reply:
I think it's better to write full function description. It's  more readable.

Reply:
So it must be with `User` service too?)